components/admin/questDetails.tsx

```diff
@@ -56,7 +58,9 @@ const AdminQuestDetails: FunctionComponent<QuestDetailsProps> = ({
   const [customError, setCustomError] = useState<string>("");
   const { showNotification } = useNotification();
   const questId = quest?.id?.toString();
+  const boostId = quest?.boosts[0].boostId.toString();
```

Comment:
There is no boostId field. The field is .id
The correct type is Boost.

Also, as I said, please map the boosts.

Reply:
It'll fix the error message